bin/deployer_mitreid

```diff
@@ -20,6 +20,18 @@ def init_ams_agent(path):
     return PullPublish(config)
 
 
+def map_token_endpoint_value(key):
+    if key == 'client_secret_post':
+        return 'SECRET_POST'
+    elif key == 'client_secret_basic':
+        return 'SECRET_BASIC'
+    elif key == 'client_secret_jwt':
+        return 'SECRET_JWT'
+    elif key == 'private_key_jwt':
+        return 'PRIVATE_KEY'
+    elif key == 'none':
+        return 'NONE'
```

Comment:
@nikosev @linathedog @kevangel79 
> The options are client_secret_post, client_secret_basic, client_secret_jwt, private_key_jwt, and none, as described in Section 9 of OpenID Connect Core 1.0 [OpenID.Core].

So can you help clarify why we need this mapping? 
1. Does the federation registry model use the standard authentication method names defined in the OIDC core spec?
2. Does the MITREid API require different names?

Reply:
> 2. Does the MITREid API require different names?

@NicolasLiampotis Yes, the client API of MITREid uses enums for these values
Please check the [apiAddClient()](https://github.com/mitreid-connect/OpenID-Connect-Java-Spring-Server/blob/1.2.x/openid-connect-server/src/main/java/org/mitre/openid/connect/web/ClientAPI.java#L193-L229) method

Reply:
Federation registry uses the following values for the token_endpoint_auth_method field 
`"token_endpoint_auth_method":["client_secret_basic","client_secret_post","client_secret_jwt","private_key_jwt","none"]`